crates/biome_service/src/workspace/scanner.rs

```diff
@@ -203,7 +233,8 @@ struct ScanFolderResult {
 /// Returns the duration of the process and the evaluated paths.
 #[instrument(level = "debug", skip(ctx))]
 fn scan_folder(folder: &Utf8Path, ctx: ScanContext) -> ScanFolderResult {
-    let start = Instant::now();
+    let start = web_time::Instant::now();
```

Comment:
`web_time::Instant` is a drop-in replacement of `std::time::Instant` backed by `performance.now()` with wasm-bindgen.